__test__/groups.test.ts

```diff
@@ -498,34 +497,48 @@ describe('getMembersFromGroups', () => {
         const groupNames = ['SDSU', 'IUSM'];
 
         // Mock GROUPS_MAP in your test environment if needed
-        jest.mock('../src/main/groups', () => ({
+        jest.mock('../src/main/propertiesService', () => ({
             loadMapFromScriptProperties: jest.fn().mockReturnValueOnce(MOCK_GROUPS_MAP)
         }));
 
+        const { getMembersFromGroups } = require("../src/main/groups");
+
```

Comment:
Because we are using dynamic module mocking, the function being tested has to be dynamically imported after all mocks for this test have been setup